pyaedt/modules/MeshIcepak.py

```diff
@@ -697,18 +697,61 @@ def add_priority(self, entity_type, obj_list=None, comp_name=None, priority=3):
             self._priorities_args.append(prio)
             args += self._priorities_args
         elif entity_type == 2:
-            prio = [
-                "NAME:PriorityListParameters",
-                "EntityType:=",
-                "Component",
-                "EntityList:=",
-                comp_name,
-                "PriorityNumber:=",
-                i,
-                "PriorityListType:=",
-                "3D",
-            ]
-            self._priorities_args.append(prio)
+            o = self.modeler.user_defined_components[comp_name]
+            if (all(part.is3d for part in o.parts.values()) is False) and (
```

Comment:
I see multiple codecov warning like `Added lines #L700 - L701 were not covered by tests`. But these lines should be covered by the new test that @nnetake added.

@Samuelopez-ansys I think those warning are a codecov error as in the codecov summary, it says that there are 9 new lines and 9 hits. Or am I missing something?

Reply:
@lorenzovecchietti 

This message is added at the beginning by codecov bot, but it could change after the Unit tests have passed, then you should just check that the Codecov action is more than 85%.